docs/user/resources/06-20-compassconnection.md

```diff
@@ -0,0 +1,93 @@
+# CompassConnection
+
+The `compassconnections.compass.kyma-project.io` CustomResourceDefinition (CRD) 
```

Comment:
Are you sure about that? The [compassConnection yaml file](https://github.com/kyma-project/kyma/blob/main/installation/resources/crds/compass-runtime-agent/compass-connection.crd.yaml#L7) says otherwise.

Reply:
It is minor change so I will not argue about that. In fact in all CRDs are defined as collections. But from the user perspective there will be always a single connection in the cluster created and managed by the  Runtime Agent.